src/index.js

```diff
@@ -471,6 +471,9 @@ function Carotte(config) {
 
         options = parseSubscriptionOptions(options, qualifier);
 
+        const transport = options.transport === null
+            ? emptyTransport
+            : (options.transport || config.transport);
```

Comment:
I would suggest to move the responsibility of defining the transport to `parseSubscriptionOptions` by adding config in the mix:

```suggestion
        options = parseSubscriptionOptions(options, qualifier, config);
```

Reply:
updated ✅